src/components/LocalIdentityModal/LocalIdentityModal.js

```diff
@@ -14,34 +14,32 @@ import IdentityBadgeWithNetwork from '../IdentityBadge/IdentityBadgeWithNetwork'
 import keycodes from '../../keycodes'
 import { EthereumAddressType } from '../../prop-types'
 
-const LocalIdentityModal = React.memo(
-  ({ opened, address, label, onCancel, onSave }) => {
-    return (
-      <Modal visible={opened} onClose={onCancel}>
-        <LocalModal
-          address={address}
-          label={label}
-          onCancel={onCancel}
-          onSave={onSave}
-        />
-      </Modal>
-    )
-  }
-)
+const LocalIdentityModal = React.memo(function LocalidentityModal({
+  opened,
+  onCancel,
+  ...props
+}) {
+  return (
+    <Modal visible={opened} onClose={onCancel}>
+      <LocalModal onCancel={onCancel} {...props} />
+    </Modal>
+  )
+})
 
 LocalIdentityModal.propTypes = {
   opened: PropTypes.bool.isRequired,
   address: EthereumAddressType,
   label: PropTypes.string,
   onCancel: PropTypes.func.isRequired,
+  onDelete: PropTypes.func.isRequired,
   onSave: PropTypes.func.isRequired,
 }
 
-function LocalModal({ address, label, onCancel, onSave }) {
+function LocalModal({ address, label, onCancel, onDelete, onSave }) {
```

Comment:
What would you think of renaming this component with the name of the file (`LocalIdentityModal`), and move the current `LocalIdentityModal` at the end of the file, as an anonymous function?

Reply:
Let's do it!